.github/workflows/go-build-test.yml

```diff
@@ -15,30 +15,102 @@ on:
       - 'go.sum'
 
 jobs:
-  build-and-test:
+  build-and-test-multi-arch:
     runs-on: ubuntu-latest
+    name: Build on ${{ matrix.arch }}
+    # Run steps on a matrix of 2 arch.
+    strategy:
+      matrix:
+        arch:
+          - x64
+          - aarch64
     steps:
-      - uses: actions/checkout@v3
-
-      - name: Set up Go
-        uses: actions/setup-go@v3
-        with:
-          go-version: '1.18'
-          check-latest: true
-
-      - name: Build UI
-        run: make ui/build
-
-      - name: Test
-        run: make go/test
-
-      - name: Build
-        run: make build
-
-      - name: Archive generatated artifacts
-        uses: actions/upload-artifact@v3
-        with:
-          name: parca-bin
-          if-no-files-found: error
-          path: |
-            bin
+    - uses: actions/checkout@v3
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: '1.18'
+        check-latest: true
+
+    - name: Build UI
+      if: ${{ matrix.arch == 'x64' }}
+      run: |
+        make ui/build
```

Comment:
What do we do in these make actions that are specific to arch?

Reply:
We have strategy defined [here](https://github.com/parca-dev/parca/pull/1060/files#diff-02549322dd3f50a460caa5562cd8842ab35f0f93ce890f824242bab7c338300bR22-R26). It takes list of architecture. Currently we are testing for `x86_64` and `aarch86`.